indexer/packages/postgres/src/db/migrations/migration_files/20240913142157_change_affiliate_info_decimal_precision_and_add_total_referred_volume.ts

```diff
@@ -0,0 +1,22 @@
+import * as Knex from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.alterTable('affiliate_info', (table) => {
+    // null indicates variable precision whereas not specifying will result in 8,2 precision,scale
+    table.decimal('affiliateEarnings', null).alter();
+    table.decimal('totalReferredFees', null).alter();
+    table.decimal('referredNetProtocolEarnings', null).alter();
```

Comment:
what does variable precision mean? Thinking about this more, we probably want scale of 6 for all fees, thats the smallest unit of USDC

Reply:
For variable precision it will not append any 0s after the decimal. Eg. 2 stays 2 and 2.123 stays 2.123. Variable precision allows for up to the maximum postgres limit.

If you set the precision to 6. It will pad 0s so 2 -> 2.000000. 

We can do it either way, the main change will be that our unit tests use the string conversion so we have to change those values. I prefer to stick with variable, as that is convention other columns in our tables use.